yawl/lib/yawl/widget/Histogram.lua

```diff
@@ -0,0 +1,146 @@
+local class = require("libClass2")
+local Frame = require("yawl.widget.Frame")
+local gpu = require("component").gpu
+
+--[[
+    Ideally, Histogram will have horizontal scrolling. Though there is some contention about the direction.
+]]
+---@class Histogram:Frame
+---@field parent Frame
+---@operator call:Histogram
+---@overload fun(parent:Frame,x:number,y:number)
+local Histogram = class(Frame)
+
+---Comment
+---@return Histogram
+---@param parent Frame
+---@param x number
+---@param y number
+function Histogram:new(parent, x, y, maxColumns) --Histogramset can be string too
+    checkArg(1, parent, "table")
+    checkArg(1, maxColumns, "number", "nil")
+    local o = self.parent(parent, x, y)
+    setmetatable(o, {__index = self})
+    o._maxColumns = maxColumns
```

Comment:
make a getter/setter methods for this member. It might be useful to be able to resize it

Reply:
not yet implemented, will do later